Rakefile

```diff
@@ -22,4 +22,19 @@ end
 desc "Run all specs"
 task :spec => 'spec:all'
 
+namespace :docker do
+  def tag
+    label = `git rev-parse HEAD`.strip
+    "dantoml/cocoapods-search-api:#{label}"
```

Comment:
I assume this is on the docker hub, this should really be under a cocoa pods account so it can be shared too 👍